docs/monitoring.md

```diff
@@ -273,8 +309,8 @@ for a running application, at `http://localhost:4040/api/v1`.
     <td>Download the event logs for all attempts of the given application as a zip file</td>
   </tr>
   <tr>
-    <td><code>/applications/[app-id]/[attempt-id]/logs</code></td>
-    <td>Download the event logs for the specified attempt of the given application as a zip file</td>
+    <td><code>/applications/[app-id]/logs</code></td>
+    <td>Download the event logs for the application as a zip file</td>
```

Comment:
OK, now i'm confusing, because the docs at the bottom say appId is `appid/attemptid` in yarn, which is true for every other element, which was why I adjusted this one. 

I'll try and clarify things more about appId vs attemptId, pull the details to the top and then have special mention here. As clearly it managed to confuse me.